config/nginx/nginx.conf

```diff
@@ -57,6 +37,14 @@ http {
     ssl_ciphers 'EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH';
     ssl_dhparam /etc/nginx/ssl/dhparam.pem;
 
+    resolver 127.0.0.11 valid=10s;
```

Comment:
IIRC this is loaded into an openresty setup. If that's the case it's possible to use `resolver local=on` here so it's not pinned to docker DNS only. [See here for more](https://github.com/openresty/openresty/#resolvconf-parsing).

Reply:
nice, thanks. I was wondering whether there was a way to get that functionality.

Reply:
I'll thoroughly test this and then add it in a subsequent merge.